packages/hub/searchers.js

```diff
@@ -72,7 +74,7 @@ class Searchers {
     return this.get(session, this.controllingBranch.name, type, id);
   }
 
-  async search(session, branch, query) {
+  async search(session, branch, query, additionalIncludes) {
```

Comment:
This is probably the root of your problem. The includes are already represented inside `query`, there's no need to change the signature here.